mne/io/eeglab/eeglab.py

```diff
@@ -243,7 +275,7 @@ class RawEEGLAB(_BaseRaw):
     """
     @verbose
     def __init__(self, input_fname, montage, preload=False, eog=(),
-                 verbose=None):
+                 event_id=dict(), event_id_func='strip-to-int', verbose=None):
```

Comment:
just an observation, our constructor isn't uniform. it should be `__init__(self, input_fname, montage, eog, misc, ..., preload, verbose)`